### PR TITLE
HRT-306 - Login flow

### PR DESCRIPTION
**Pull request by gdbarnes**
Should allow user to save and exit, and then log back in to an incomplete application using their email address after verification.

### CODE REVIEW

File: pages/apply/sign-in.tsx [hunk 1]
```diff
@@ -22,16 +23,20 @@ const ApplicationSignInPage = (): JSX.Element => {
   }, [isLoggedIn]);
 
   const onSubmit = async (values: FormData) => {
-    // TODO create a verify code based on email provided and send to verify page
-    //dispatch(createVerifyCode(values.emailAddress));
-    //router.push('/apply/verify');
+    dispatch(createVerifyCode(values.emailAddress));
```
Comment (tcmorris) [inline pages/apply/sign-in.tsx:26]: Add try/catch with `setUserError`

File: pages/apply/start.tsx [hunk 2]
```diff
@@ -41,19 +34,12 @@ const ApplicationStartPage = (): JSX.Element => {
             nationalInsuranceNumber: values.nationalInsuranceNumber,
```
Comment (tcmorris) [inline pages/apply/start.tsx:34]: Think `updateBeforeFirstSave` might need to be updated to `updateApplicant`

**Review by tcmorris — COMMENTED** [review]
Changes look ok to me, have tested briefly but haven't covered both scenarios.

File: pages/apply/sign-in.tsx (hunk 1 above)
Comment (gdbarnes) [reply pages/apply/sign-in.tsx]: try/catch added

File: pages/apply/start.tsx (hunk 2 above)
Comment (gdbarnes) [reply pages/apply/start.tsx:41]: I've put this in but haven't been able to run a true new application. We might need to ensure that the main applicant's id is available at this stage.

**Pull request merged**